client/src/App.js

```diff
@@ -21,8 +23,10 @@ export default class App extends Component {
   render() {
     return (
       <div>
-        <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Link to="/"><SavedList list={this.state.savedList} /></Link>
+        {/* <div>Replace this Div with your Routes</div> */}
+        <Route exact path="/" component={MovieList} />
+        <Route path="/movies/:id" component={Movie} render={(props)=> <Movie {...props} />} />
```

Comment:
remove the component={Movie} you are using render and creating the component there